src/constructors.ts

```diff
@@ -0,0 +1,68 @@
+import { Result, ResultValues, AggregateResponse, Results } from "./results";
+
+/**
+ * valueFromProto unpacks a proto value.
+ */
+export const valueFromProto = (proto: any): any => {
```

Comment:
we know the return type of this [`string | string[] | null`](https://github.com/sajari/sdk-node/blob/4f37e2169608c1e6f3bb6a4e84c61925b67f868e/src/utils.ts#L8)

Reply:
👍